src/components/CommentInput/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { StyleSheet, TextInput, View, ViewStyle } from 'react-native';
+
+import { useInput } from '../../utils/hook';
+
+interface Props {
+	style?: ViewStyle;
+	containerStyle?: ViewStyle;
+}
+
+export const CommentInput: FC = ({
+	style = {},
+	containerStyle = {},
+}: Props) => {
+	const input = useInput('');
+	return (
+		<View style={[styles.container, containerStyle]}>
+			<TextInput
+				multiline
+				numberOfLines={6}
+				style={[styles.input, style]}
+				placeholder="Write your comment..."
+				placeholderTextColor="rgba(255, 255, 255, 0.2)"
+				{...input}
+			/>
+		</View>
+	);
+};
+
+export default CommentInput;
+
+const styles = StyleSheet.create({
+	container: {
+		width: '100%',
```

Comment:
not really sure if `width: '100%'` is useful?

Reply:
it's not useful, i've remove it